mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -13,7 +13,7 @@
 					{
 						id = 2,
 						type = "description",
-						text = ::Reforged.Mod.Tooltips.parseString("Characters gain experience as they or their allies slay enemies in battles. If a character has accumulated sufficient experience, he\'ll [level up|Concept.Level] and be able to increase his [attributes|Concept.CharacterAttribute] and pick a [perk|Concept.Perk] that grants a unique bonus.\n\nBeyond the " + ::Const.XP.MaxLevelWithPerkpoints + "th  character level, characters are veterans and gain perk points only every 4th level. They can still improve their attributes but the attribute gain per level is small.")
+						text = ::Reforged.Mod.Tooltips.parseString("Characters gain experience as they or their allies slay enemies in battles. If a character has accumulated sufficient experience, he\'ll [level up|Concept.Level]. Experience for killing an entity is shared among all the characters who did damage to that entity, proportional to the damage dealt by each character. Some experience is also awarded to all members of a faction who did damage to an entity, proportional to the damage dealt by that faction. \n\nBeyond the " + ::Const.XP.MaxLevelWithPerkpoints + "th  character level, characters are veterans and gain perk points only every 4th level. They can still improve their attributes but the attribute gain per level is small.")
```

Comment:
That new section is two mouthful to parse.
Also it should have the global constant about the exact percentage of the shares built-in.

I'm not happy with the following variant but maybe it can get us closer to an acceptable end-result.

The amount of experience that is gained from deaths is proportional to the amount of damage your company dealt to it. `::MSU.Text.colorizePercentage(::Const.XP.XPForKillerPct * 100.0)` of that experience is shared proportionally to the contributor. The other `::MSU.Text.colorizePercentage(100 - ::Const.XP.XPForKillerPct * 100.0)` are shared equally among all characters of that faction.